content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/expiration_dates.rst

```diff
@@ -1,65 +1,266 @@
+Set expiration dates on manufactured products
+=============================================
 
-.. image:: expiration_dates/expiration_dates_03.png
+Expiration dates can also be generated for products manufactured in-house. To assign expiration
+dates to manufactured products, a manufacturing order (MO) needs to be completed.
+
+To create a :abbr:`MO (manufacturing order)`, go to :menuselection:`Manufacturing app --> Operations
+--> Manufacturing Orders`, and click :guilabel:`Create`. Choose a product to manufacture from the
+:guilabel:`Product` field drop-down menu, then select the :guilabel:`Quantity` to produce.
+
+.. image:: expiration_dates/expiration-dates-manufacturing-order.png
    :align: center
+   :alt: Manufacturing order for product with expiration date.
+
+.. note::
+   To manufacture a product, there must be materials to consume in the :guilabel:`Product` lines.
+   This can be achieved either by creating a :guilabel:`Bill of Material` for the
+   :guilabel:`Product`, or manually adding materials to consume by clicking :guilabel:`Add a line`.
+
+Once ready, click :guilabel:`Confirm`.
+
+Next to :guilabel:`Lot/Serial Number`, either select an existing lot number from the drop-down menu,
+or click the green :guilabel:`+ (plus)` sign to automatically assign a new lot number.
+
+Then, select a number of units for the :guilabel:`Quantity` field, and click :guilabel:`Mark as
+Done`.
+
+Click on the :guilabel:`External Link` icon in the assigned :guilabel:`Lot/Serial Number` field. A
+pop-up appears, revealing a detail form for that specific number.
+
+On that pop-up, under the :guilabel:`Dates` tab, all expiration information that was previously
+configured for the product is displayed. That same information is also available on the detail form
+for that specific product, or by going to :menuselection:`Inventory app --> Products --> Lots/Serial
+Numbers`.
 
-.. image:: expiration_dates/expiration_dates_04.png
+.. image:: expiration_dates/expiration-dates-dates-tab-lot-number.png
    :align: center
+   :alt: Dates tab with expiration information for specific lot number.
+
+Sell products with expiration dates
+===================================
+
+Selling perishable products with expiration dates is done the same as any other type of product. The
+first step in selling perishable products is to create a sales order.
+
+To do that, go to :menuselection:`Sales app --> Create` to create a new quotation, and fill out the
+information on the sales order form.
+
+Add a :guilabel:`Customer`, click :guilabel:`Add a product` to add the desired products to the
+:guilabel:`Product` lines, and set a :guilabel:`Quantity` for the products.
+
+Then, click the :guilabel:`Other Info` tab. Under the :guilabel:`Delivery` section, change the
+:guilabel:`Delivery Date` to a date after the expected date, and click the :guilabel:`green
+checkmark` to confirm the date. Finally, click :guilabel:`Confirm` to confirm the sales order.
```

Comment:
Suggestions

1. specify that it's an icon
```suggestion
:guilabel:`Delivery Date` to a date after the expected date, and click the :guilabel:`green
checkmark` icon  to confirm the date. Finally, click :guilabel:`Confirm` to confirm the sales order.
```

2. ...or use a standard emoji (~= v1.0)
```suggestion
:guilabel:`Delivery Date` to a date after the expected date, and click the :guilabel:`✅
(checkmark)` icon to confirm the date. Finally, click :guilabel:`Confirm` to confirm the sales order.
```